packages/application-shell/src/components/application-shell-authenticated/application-shell-authenticated.tsx

```diff
@@ -1,3 +1,4 @@
+// NOTE: this file contains feature flag that will be removed once the new navigation is enabled for all users
```

Comment:
We're not actually using any specific feature flag here so maybe what we can to is to add a comment when using the `NewMainNavigationFlagWrapper` component about it is intended to be temporary.

Reply:
Oh, yeah, makes sense. Thanks! Added here - [3cacd07](https://github.com/commercetools/merchant-center-application-kit/pull/3159/commits/3cacd07554402eeb33b3d79360ed7dc6d4ded311).

Reply:
What I meant it to move this comment to where we use the new temporary component, but that's just a suggestion.

Reply:
Oh, got you now. Updated here - [79547ef](https://github.com/commercetools/merchant-center-application-kit/pull/3159/commits/79547eff77c1aee9b15b1d892fac0de0bb477abe).